config/schedule.rb

```diff
@@ -18,3 +18,16 @@
 # end
 
 # Learn more: http://github.com/javan/whenever
+
+log_output_path = ENV["EXPORT_SERVICE_CRON_LOG_OUTPUT_PATH"] || "/srv/ruby/flood-risk-back-office/shared/log/"
+set :output, File.join(log_output_path, "whenever_cron.log")
+set :job_template, "/bin/bash -l -c 'eval \"$(rbenv init -)\" && :job'"
```

Comment:
@Cruikshanks I copied this from WEX, but I dunno if this is going to be correct in production.

Reply:
That should be fine @cintamani 😁